lib/ibm_watson/visual_recognition_v3.rb

```diff
@@ -226,7 +226,7 @@ def detect_faces(images_file: nil, url: nil, images_file_content_type: nil, imag
     #########################
 
     ##
-    # @!method create_classifier(name:, **args)
+    # @!method create_classifier(name:, positive_examples:, negative_examples: nil, positive_examples_filename: {}, negative_examples_filename: nil)
```

Comment:
`positive_examples: nil`?

Reply:
actually that should not have nil for the default value because it's a required parameter